frameworks/autogen/chess-game/agentuity_agents/ChessGame/agent.py

```
@@ -0,0 +1,189 @@
+                self.board.push(move)
+                self.move_history.append(move_uci)
+                
+                if self.board.is_checkmate():
+                    self.game_over = True
+                    self.winner = "White" if self.board.turn == chess.BLACK else "Black"
+                elif self.board.is_stalemate() or self.board.is_insufficient_material():
+                    self.game_over = True
+                    self.winner = "Draw"
+                    
+                return True
+            return False
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)